roles/letsencrypt/templates/renew-certs.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I think we can do `#!/usr/bin/env python{{ ansible_python_version[0] }}`

Then it would output either `python2` or `python3`.

Reply:
This is working from my testing

Reply:
Here are all the Python binaries installed on 16.04:

```
python      python2     python2.7   python3     python3.5   python3.5m  python3m
```

`python2` is just `python2.7` which also happens to just be plain `python`.

That being said... I'm pretty sure all Trellis 16.04 installs get Python 3 installed as well (not explicitly but as a transitive dependency).

So I'm wondering if this is worth it or if we should just default to `python3` as you had it.

Reply:
Let's just leave this as you had it for now. Keep it simpler unless we need to make it more flexible.